src/components/BlogResourceItem.tsx

```diff
@@ -19,24 +21,37 @@ interface Blog {
     img?: Image
 }
 
-export const BlogResourceItem: FunctionComponent<Props> = ({ blog }) => (
-    <div className="w-100 row mx-0 mb-5 pb-4 border-bottom">
-        <div className="col-sm-8 col-md-9">
-            <a
-                className="posts-list__post-title-link"
-                href={blog.href}
-                title={blog.title}
-                data-button-style={buttonStyle.text}
-                data-button-location={buttonLocation.body}
-                data-button-type="cta"
-            >
-                <h3 className="font-weight-bold">{blog.title}</h3>
-            </a>
-            <p className="text-gray-5">{blog.type}</p>
-            <p>{blog.description}</p>
-        </div>
-        <div className="col-sm-4 col-md-3 d-flex align-items-center">
-            {blog.img && <img className="flex-1 w-100" alt={blog.img.alt} src={blog.img.src} />}
+export const BlogResourceItem: FunctionComponent<Props> = ({ blog }) => {
+    const defaultThumbnails: string[] = ['darkNebulous1Sm', 'darkNebulous2Sm', 'darkNebulous3Sm', 'darkNebulous4Sm']
+    const thumbnail: string = defaultThumbnails[Math.floor(Math.random() * 4)].toString()
+    const variant: Background['variant'] = thumbnail as Background['variant']
```

Comment:
This creates a random thumbnail if no img is provided for `BlogResourceItem`. I figured this would be better than not having an image at all and gives some differentiation between posts.